penguin/integrations/mcp/server_tools/ituv.py

```
@@ -0,0 +1,619 @@
+        task_id = _required_str(arguments, "task_id")
+        dry_run = _optional_bool(arguments.get("dry_run"), True)
+        task = project_manager.get_task(task_id)
+        if task is None:
+            return {"error": "task_not_found", "task_id": task_id}
+        artifact = ArtifactEvidence(
+            key=_required_str(arguments, "key"),
+            kind=_required_str(arguments, "kind"),
+            path=_optional_str(arguments.get("path")),
+            producer_task_id=_optional_str(arguments.get("producer_task_id")) or task_id,
+            created_at=_optional_str(arguments.get("created_at"))
+            or datetime.utcnow().isoformat(),
```

Ruff (0.15.12)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)